docs/copilot/workspace-context.md

```diff
@@ -0,0 +1,72 @@
+---
+Order: 7
+Area: copilot
+TOCTitle: Workspace Context
+ContentId: c77dcce9-4ba9-40ac-8ae5-2df855088090
+PageTitle: Chat using @workspace Context References
+DateApproved: 02/28/2024
+MetaDescription: Overview of how the @workspace Copilot chat participants works and how it manages context.
+---
+# How the @workspace Copilot chat participant works
+
```

Comment:
Should we first have a sentence or two intro on what participants are?

Reply:
Afaik we should not open the H1 with chat participant, and optimize for SEO and a more natural language